js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
```

Comment:
I think that's risky, because this could happen because of some issue and might lead to misleading bug reports.

You should be able to get the value of `enable_previews` via OC javascript methods.